event-streams/src/main/java/com/redhat/cloud/platform/playbook_dispatcher/RunEventTransform.java

```diff
@@ -143,6 +145,10 @@ Payload buildRunPayload(Struct input) {
         payload.setTimeout(input.getInt32("timeout"));
         payload.setCreatedAt(input.getString("created_at"));
         payload.setUpdatedAt(input.getString("updated_at"));
+        
+        if (input.get("org_id") != null) {
```

Comment:
hey @CodyWMitchell wrt this field being optional - it may be better to implement the subtasks in order:

1. make sure all new db records have org_id set
2. migrate old records to have org_id set
3. once it's safe to assume org_id is set for all records, do the rest of the subtasks, such as the event stream transformation

Reply:
I think I'll need [RHCLOUD-17719](https://issues.redhat.com/browse/RHCLOUD-17719) to handle the db updates. It is currently in progress.